elastic/kibana
# [Fleet] updated offline query to be consistently 10 intervals

## Summary

Change left out from first pr.
Fixes https://github.com/elastic/kibana/issues/134182

Expect offline agent badge to include agents that are not updated in 5m (`last_checkin`)

To verify:
- enroll an agent, wait for it to be healthy, unenroll it
- observe that agent is still online after 2 mins, but goes offline after 5 mins
- verify that offline badge count is consistent with the agent list's status

<5 minutes - online
<img width="1001" alt="image" src="https://user-images.githubusercontent.com/90178898/175313243-b38510ee-8307-481d-b6ce-bc9f9bf1d2d6.png">

>5 minutes - offline
<img width="1010" alt="image" src="https://user-images.githubusercontent.com/90178898/175313696-597b05aa-4d52-4a4d-a4af-ed420d4ab2db.png">


### Checklist
